protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -146,15 +146,7 @@ service TradingDataService {
   //
   // Get a list of the changes in account balances over a period of time.
   //
-  // An account is defined as a set of asset_id, type, party_id, and market_id.
-  // - Every account has an associated asset and type.
-  // - Certain account types such as the global reward pool for example, do not have an associated party.
-  //   These are denoted by the special party ID 'network'
-  // - Certain account types do not have an associated market such as the general party accounts, for example.
-  //   These are denoted by the special market ID '' (the empty string)
-  //
-  // `ListBalanceChangesRequest` will return a list of `AggregatedBalance` records,
-  // with a row for each block at which a given account's balance changes.
+  // `ListBalanceChangesResponse` will return a paged list of `AggregatedBalance` records.
```

Comment:
I'm going to suggest we remove this line because we can't _really_ reference proto structs like this because that isn't necessarily how they look from the REST API docs. 
```suggestion

```

Reply:
Yes, I removed the line now, thanks!